react-app-drawer-component/justin-app/src/AppDrawer.js

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import { FiMenu } from 'react-icons/fi';
+import './AppDrawer.css';
+
+export default function AppDrawer({ items }) {
+  const [menu, setMenu] = useState(false);
```

Comment:
It's unclear what this is used for? I can read the rest of the code and figure it out, but how is "menu" a boolean? Maybe isOpen or menuIsShown or something along those lines would be better?